stdlib/asm/crypto/hashes/native.masm

```diff
@@ -1,3 +1,20 @@
+#! Given the hasher state, returns the hash output
+#!
+#! Input: [C, B, A, ...]
+#! Ouptut: [HASH, ...]
+#! Where: For the native RPO hasher HASH is B.
+#! Cycles: 9
+export.digest_output
```

Comment:
I would probably name it `state_to_digets` or `get_digest_from_state` or something similar.

Reply:
yep, that is better! changed